### PR TITLE
ticdc: add new rules and information for syncpoint 

### PR DESCRIPTION
**Pull request by hongyunyan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/17710
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-upstream-downstream-check.md [hunk 1]
```diff
@@ -95,7 +95,12 @@ Here is a configuration example of the `Datasource config` section:
...
+- Before TiCDC creates a changefeed, make sure that the value of the TiCDC configuration item `enable-sync-point` is set to `true`. Only in this way, Syncpoint is enabled and the `ts-map` is saved in the downstream. The format of the configuration item `sync-point-interval` is `"h m s"`, for example `"1h30m30s"`, and the minimum value is `"30s"`. For more complete configuration information, please refer to [TiCDC task configuration file](/ticdc/ticdc-changefeed-config.md).
 - When you perform data validation using Syncpoint, you need to modify the Garbage Collection (GC) time of TiKV to ensure that the historical data corresponding to snapshot is not collected by GC during the data check. It is recommended that you modify the GC time to 1 hour and recover the setting after the check.
 - The above example only shows the section of `Datasource config`. For complete configuration, refer to [sync-diff-inspector User Guide](/sync-diff-inspector/sync-diff-inspector-overview.md).
 - Since v6.4.0, only the changefeed with the `SYSTEM_VARIABLES_ADMIN` or `SUPER` privilege can use the TiCDC Syncpoint feature.
```
Comment (lilin90) [inline ticdc/ticdc-upstream-downstream-check.md:101]:
```suggestion
- Starting from v6.4.0, only the changefeed with the `SYSTEM_VARIABLES_ADMIN` or `SUPER` privilege can use the TiCDC Syncpoint feature.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

File: ticdc/ticdc-upstream-downstream-check.md [hunk 2]
```diff
@@ -95,7 +95,12 @@ Here is a configuration example of the `Datasource config` section:
...
-- Since v6.4.0, only the changefeed with the `SYSTEM_VARIABLES_ADMIN` or `SUPER` privilege can use the TiCDC Syncpoint feature.
+- Starting from v6.4.0, only the changefeed with the `SYSTEM_VARIABLES_ADMIN` or `SUPER` privilege can use the TiCDC Syncpoint feature.
+- Starting from v8.2.0, TiCDC makes the following adjustments to the generation rules of `primary_ts` values:
+    - Whenever TiCDC generates a new `primary_ts`, its value must be an integer multiple of `sync-point-interval`.
+    - For each new changefeed, TiCDC calculates an initial `primary_ts`. This initial value is greater than or equal to the changefeed start time (`startTs`), and is the smallest integer multiple of `sync-point-interval`.
+
+  This setting is used to align the Syncpoints of different changefeeds during data synchronization. For example, multiple downstream clusters can restore to the `secondary_ts` of a Syncpoint with the same `primary_ts` by executing the [`FLASHBACK TABLE`](/sql-statements/sql-statement-flashback-table.md) statement, achieving data consistency across downstream clusters.
```
Comment (hfxsd) [inline ticdc/ticdc-upstream-downstream-check.md:106]:
```suggestion
  This setting is used to align the Syncpoints of different changefeeds during data replication. For example, multiple downstream clusters can restore to the `secondary_ts` of a Syncpoint with the same `primary_ts` by executing the [`FLASHBACK TABLE`](/sql-statements/sql-statement-flashback-table.md) statement, achieving data consistency across downstream clusters.
```

**Review by hfxsd — APPROVED** [review]

**Review by flowbehappy — APPROVED** [review]

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**